src/BUILD.bazel

```diff
@@ -129,6 +128,21 @@ ts_devserver(
     deps = ["//src"],
 )
 
+# This list is auto-updated by /tools/generator
```

Comment:
github is showing a bunch of non-printable unicode here?

Reply:
Yeah, I noticed these 0x1c bytes as well. I don't know where they come from. I'll investigate.